lib/internal/Magento/Framework/DB/Logger/Monolog.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\DB\Logger;
+
+use Magento\Framework\Debug;
+use Psr\Log\LoggerInterface;
+use Magento\Framework\Serialize\SerializerInterface;
+
+/**
+ * Class Monolog
+ */
+class Monolog extends LoggerAbstract
```

Comment:
It probably shouldn't be "Monolog", as it just uses PSR logger interface. Not sure if it should be "PsrLogger" :) or something else.